devlib/module/cpufreq.py

```diff
@@ -375,9 +375,18 @@ def set_all_governors(self, governor):
         """
         Set the specified governor for all the (online) CPUs
         """
-        return self.target._execute_util(
+        try:
+            return self.target._execute_util(
                 'cpufreq_set_all_governors {}'.format(governor),
                 as_root=True)
+        except TargetError:
```

Comment:
The `TargetError`  may not necessarily have been cause by an unsupported governor (e.g. someone may have pulled a cable from the device). It's worth checking the  error output to make sure that the wrong governor is the cause.